src/config/dynamic-theme-fixes.config

```diff
@@ -13294,9 +13294,15 @@ img.act-icon-dark-gray
 [src="/chrome/static/images/google-translate/screen-english.png"]
 
 CSS
-.RNNXgb,
+.minidiv .sfbg,
+#searchform.minidiv,
+.minidiv .emcav .RNNXgb .S6VXfe,
+.CvDJxb .RNNXgb,
 .aajZCb {
-    box-shadow: 0 0 2px 0 ${rgba(0,0,0,0.16)}, 0 0 0 1px ${rgba(0,0,0,0.08)} !important;
+    box-shadow:0 1px 6px 0 rgba(0,0,0,.28) !important;
+}
+.RNNXgb:hover {
+    box-shadow:0 1px 6px 2px rgba(0,0,0,.28) !important;
```

Comment:
Same thing, don't think making it darker is a good idea. It's with my settings(slightly higher contrast) already un-noticeable.

Reply:
Here, perhaps stronger shadows could be with 2px blur and 5px spread
`box-shadow:0 1px 2px 5px ${rgba(255,255,255,.28)} !important;`

Reply:
![image](https://user-images.githubusercontent.com/25481501/123088446-41e72080-d415-11eb-8c94-018f1f9a45ab.png)
It's the same un-noticeable dark for me.

Reply:
I see... So I'll close this pull request, then?

Reply:
Pretty much, I don't really see any fixes from this PR.